js/components/visualizations/TopPagesCircleGraph.js

```diff
@@ -136,7 +135,7 @@ function TopPagesCircleGraph({ dataHrefBase }) {
     return transformedData;
   }
 
-  function drawChart(transformedData) {
+  async function drawChart(transformedData) {
```

Comment:
```suggestion
  function drawChart(transformedData) {
```

You can correct me but I don't see the need for this function to be asynchronous.